mono/mini/mini-generic-sharing.c

```diff
@@ -1208,6 +1209,8 @@ instantiate_info (MonoDomain *domain, MonoRuntimeGenericContextInfoTemplate *oti
 		g_assert (info->klass->vtable);
 		method = info->klass->vtable [ioffset + slot];
 
+		method = mono_class_inflate_generic_method_checked (method, context, &error);
```

Comment:
This call can fail and if it does, we'll get a crash with no information since we're not checking the result.